tools/cli/commands/changelog.js

```diff
@@ -834,44 +823,56 @@ async function promptChangelog( argv, needChangelog ) {
  */
 async function changelogAddPrompt( argv, needChangelog, uniqueProjects ) {
 	const totalProjects = [ ...needChangelog, ...uniqueProjects ];
+
+	// If only one project needs a changelog, confirm.
```

Comment:
There's some redundancy in the code below. You might want to restructure it something like this

```js
let prompts;

if ( totalProjects.length === 1 ) {
    prompts = { /* changelogConfirm: Add changelog for the one project? */ };
} else if ( defaultProjects.length <= 1 && uniqueProjects.length > 0 ) {
    prompts = { /* separateChangelogFiles: Found N projects needing changelogs. Create for each one? */ };
} else if ( defaultProjects.length > 0 && uniqueProjects.length === 0 ) {
    prompts = {
        /* sameChangelog: Found N projects. Same to all? */
        /* separateChangelogFiles: (else) For each individually? */
    };
} else {
    prompts = {
        /* sameChangelog: Found N projects that can use the same changelog and M that need individual ones. Same to the N? */
        /* separateChangelogFiles: (else) For each individually? */
    };
}

const response = await inquirer.prompt( prompts );

// Return the object only if some question was answered "yes", otherwise false.
return Object.values( response ).some( v => v ) ? response : false;
```

That also (in combination with my suggestion on line 388) should happen to fix some weird behavior in a case like `touch projects/packages/changelogger/foo.txt projects/plugins/jetpack/foo.txt && git add . && jetpack changelog add`
```
Checking if changelog files are needed. Just a sec...
? Found 1 projects that can accept the same changelog file.
  Found 1 project(s) that requires a separate changelog configuration. 
  Add same changelog file to 1 project(s)? No
? Create changelog for each project individually?
```
Just treat them all as unique.